pulsar-client-cpp/test-conf/authentication.conf

```diff
@@ -0,0 +1,265 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Sure, I just copied the files used in the unit tests executed from Travis, but it makes sense to fix the naming